backend/protos/mgmt/v1alpha1/user_account.proto

```diff
@@ -226,12 +226,31 @@ message IsAccountStatusValidRequest {
   // An optional count of records to be added to the current usage for validation.
   optional uint64 requested_record_count = 2;
 }
+
 message IsAccountStatusValidResponse {
   bool is_valid = 1;
   // If the account is not valid, a reason for why may be provided.
   optional string reason = 2;
   // Whether or not the process should decide to continue polling for validitiy updates
   bool should_poll = 3;
+  // A count of the currently used records for the current billing period.
+  // This may go over the allowed record count depending on when the record count is polled by the metric system.
+  optional uint64 used_record_count = 4;
+  // The allowed record count. It will be null if there is no limit.
+  optional uint64 allowed_record_count = 5;
+  // The current status of the account. Default is valid.
+  optional AccountStatus account_status = 6;
```

Comment:
I think `used_record_count` and `account_status` can probably get away with not being optional since we have sane defaults for them.